TestAssignmentDoItSoftware/TestAssignmentDoItSoftware/EditTaskScreen/EditTaskScreenViewController.h

```diff
@@ -0,0 +1,56 @@
+//
+//  EditTaskScreenViewController.h
+//  TestAssignmentDoItSoftware
+//
+//  Created by Vladimir Samoylenko on 2/15/19.
+//  Copyright © 2019 TheRedQueen. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+#import <UserNotifications/UserNotifications.h>
+#import "AppDelegate.h"
+#import "TASingleton.h"
+#import "DateDueScreenViewController.h"
+#import "TaskDetailViewController.h"
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface EditTaskScreenViewController : UIViewController{
+    
+    NSNumber *dateDue;
+    NSMutableDictionary *dictionaryTask;
+    NSNumber *tid;
```

Comment:
Arguably it should contain the `dictionaryTask` and operate on it – having dedicated ivars per prop seems somewhat overwhelming.